src/components/Button/SystemUIButton/SystemUIButton.tsx

```diff
@@ -46,6 +47,7 @@ export const SystemUIButton: FC<ButtonProps> = React.forwardRef(
             classNames,
             styles.button,
             styles.buttonSystemUi,
+            { [styles.transparentBg]: !!transparentBg },
```

Comment:
Why do we need !! ??

Reply:
For converting undefine to boolean; since prop does not have default boolean value. Will pass the default prop value and remove this.